package.json

```diff
@@ -14,10 +14,7 @@
   "bugs": {
     "url": "https://github.com/CSSLint/csslint/issues"
   },
-  "license": {
-    "type": "MIT",
-    "url": "https://github.com/CSSLint/csslint/blob/master/LICENSE"
-  },
+  "license": "SEE LICENSE IN LICENSE",
```

Comment:
This doesn't really matter, but I'm curious why you went with the file syntax instead of "MIT"